packages/terra-site/src/examples/tag/Index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-tag/docs/README.md';
+import { version } from 'terra-tag/package.json';
+
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import TagSrc from '!raw-loader!terra-tag/src/Tag';
+
+// Example Files
+import TagDefault from './TagDefault';
+import TagIconAndText from './TagIconAndText';
+import TagHref from './HrefTag';
+
+const TagExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
+    <Markdown id="readme" src={ReadMe} />
+    <PropsTable id="props" src={TagSrc} />
+    <h2 id="default">Tag</h2>
```

Comment:
Since there is the headline Tag on the page by default, I would be explicit in this header, and say "Default Tag"

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/8ae98ee6dd52b4143445e03ba25030d29081b08b).